defaults/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
 
-# Version to install, defaulting to 5.6
+# Version to install, defaulting to 5.7
 mysql_version_major: "5"
-mysql_version_minor: "6"
+mysql_version_minor: "7"
```

Comment:
Changing the default version is a bit dangerous, because in most of our ansible hosts we don't have the minor version set in config files and it will upgrade the percona packages from 5.6 to 5.7.

This upgrade will fail, because it requires the packages purge before the upgrade.

We could add new tasks to the role to get the installed percona percona version, and abort if the major or minor version changes.

Reply:
Thank you @mamedin, this is fixed. Default version is 5.6, except for Ubuntu>=20 (in which case it is set to 5.7 because there are no 5.6 packages available)